conda_build/environ.py

```diff
@@ -965,7 +966,11 @@ def get_install_actions(
             with capture():
                 try:
                     precs = _install_actions(prefix, index, specs)["LINK"]
-                except (NoPackagesFoundError, UnsatisfiableError) as exc:
+                except (
+                    NoPackagesFoundError,
```

Comment:
We may want to use the non-aliased `ResolvePackageNotFound` here for clarification.
(`conda.exceptions`/`conda.exports` sets `NoPackagesFound = NoPackagesFoundError = ResolvePackageNotFound`.)